released/packages/coq-bignums/coq-bignums.8.7.0/opam

```diff
@@ -10,14 +10,13 @@ authors: [
   "Evgeny Makarov"
   "Pierre Letouzey"
 ]
-license: "LGPL 2"
+license: "LGPL-2.1-only"
 build: [
-  [make]
+  [make "-j%{jobs}%"]
```

Comment:
```suggestion
  [make "-j%{jobs}%" {ocaml:version >= "4.06"}]
```
@erikmd parallel builds were intentionally disabled here since they caused flaky builds with OCaml 4.05.0 - a bug that was fixed in OCaml 4.06, see discussion in AbsInt/CompCert#327. Maybe you can add the suggested fix for the other packages as well?

Reply:
Yes, I was precisely wondering about the fact the `-j` option was missing and found issue https://github.com/coq/coq/issues/10704#issuecomment-600361081, indeed.
Thanks for your suggestion, I will apply it for all related bignums versions.